packages/react-composites/src/composites/CallWithChatComposite/EmbeddedPeoplePane.tsx

```diff
@@ -62,14 +75,32 @@ export const EmbeddedPeoplePane = (props: {
         onPeopleButtonClicked={props.onPeopleButtonClick}
       >
         <Stack verticalFill tokens={peoplePaneContainerTokens}>
-          {participantList}
+          <Stack.Item>{participantList}</Stack.Item>
+          {
+            // Only render LocalAndRemotePIP when this component is NOT hidden because VideoGallery needs to have
+            // possession of the dominant remote participant video stream
+            !props.hidden && (
+              <Stack horizontalAlign="end" styles={localAndRemotePIPContainerStyles}>
+                <Stack styles={localAndRemotePIPStyles}>
+                  <LocalAndRemotePIP
+                    {...pictureInPictureProps}
+                    {...pictureInPictureHandlers}
+                    // eslint-disable-next-line @typescript-eslint/no-empty-function
```

Comment:
Instead of telling eslint to complain that we don't have a comment saying why function is empty, just move the reason inside :)

```ts
onClick={() => { /* no onClick behavior in peoplePane */ }
```

Actually, more importantly, why is there an `onClick` prop that isn't used in this case? Is this for the camera switcher? Should this be added in the people pane in a future PR? In that case, please add a TODO in the comment instead.

Reply:
Looking at this again, `LocalAndRemotePipProps.onClick` should simply be made optional.